src/app/fish/fish.component.ts

```diff
@@ -0,0 +1,22 @@
+import {Component, Input, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'sf-fish',
+  templateUrl: './fish.component.html',
+  styleUrls: ['./fish.component.scss']
+})
+export class FishComponent implements OnInit {
+
+  @Input() fish;
+  public uncaught = ['fish', 'centreMiddle', 'animated', 'bounceInUp'];
+  public caught = ['fish', 'centreMiddle', 'animated', 'shake'];
```

Comment:
Add in more fish catching animations here?

Reply:
yeah can do with more of them, need to make this part better in general as there's duplication